class Foo {
    void foo() throws Exception {}
}